### DISCUSSION TITLE
Implement .NET IL Trimming

### DISCUSSION THREAD

**Discussion by TheCodeTraveler** [CommunityToolkit/Maui.Markup#35] [New Feature Discussions]
In .NET 6.0, the team [introduced IL Trimming](https://devblogs.microsoft.com/dotnet/announcing-net-6/?WT.mc_id=mobile-0000-bramin#il-trimming).

For the .NET MAUI Community Toolkit, we should leverage the .NET team's guidelines on [Preparing .NET Libraries for Trimming](https://docs.microsoft.com/dotnet/core/deploying/trimming/prepare-libraries-for-trimming?WT.mc_id=mobile-0000-bramin).

@pictos has volunteered to lead this effort.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
